packages/tailwindcss-x/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tailwindcss-x",
+  "private": true,
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "autoprefixer": "^10.4.4",
+    "cssnano": "^5.0.15",
+    "postcss": "^8.4.12",
+    "postcss-import": "^14.1.0",
+    "tailwindcss": "^3.0.23",
+    "typescript": "~4.6.2",
+    "vite": "^2.9.0"
```

Comment:
I think we should use always the ~ and not the ^ modifier for external dependencies versions